### PR TITLE
Bump uuid to 0.8.x

### LINKED ISSUE #2204: uuid v0.8 support
uuid v0.8 was released a few days ago. It would be nice if Diesel supported that. Currently there is uuid and uuidv07 support.

### PR DESCRIPTION
**Pull request by weiznich**
Fixes #2204

### CODE REVIEW

File: diesel_tests/Cargo.toml [hunk 1]
```diff
@@ -14,12 +14,12 @@ dotenv = ">=0.8, <0.11"
 [dependencies]
 assert_matches = "1.0.1"
 chrono = { version = "0.4" }
-diesel = { path = "../diesel", default-features = false, features = ["quickcheck", "chrono", "uuid", "serde_json", "network-address", "numeric", "with-deprecated"] }
+diesel = { path = "../diesel", default-features = false, features = ["quickcheck", "chrono", "uuidv07", "serde_json", "network-address", "numeric", "with-deprecated"] }
```
Comment (weiznich) [inline diesel_tests/Cargo.toml]: I think it's more realistic to use the new uuid versions in the test.

**Comment by ForsakenHarmony** [comment]
when will this be released?

**Comment by weiznich** [comment]
With the next release. We do not give any ETA‘s other that: When it's done.
(Asking those questions will not make that happen any sooner.)

**Comment by ForsakenHarmony** [comment]
Do you have any milestones or smth one could look at too see what's required for it to be done?

**Comment by weiznich** [comment]
Alt least everything in https://github.com/diesel-rs/diesel/milestone/16. Additionally anything that seems useful and is ready till the completion of the mile stone. 
Otherwise if someone really needs a release containing the version bumps, they should backport the required changes (and everything else marked as "maybe backport") to the 1.4.x branch and open a PR. If everything is fine I may find some time to do a release, but I cannot promise anything concrete there.

**Pull request merged**